requirements/static/ci/common.in

```diff
@@ -30,6 +31,9 @@ moto>=2.0.0 ; python_version >= '3.6'
 napalm; sys_platform != 'win32' and python_version > '3.6'  and python_version < '3.10'
 paramiko>=2.10.1; sys_platform != 'win32' and sys_platform != 'darwin'
 passlib[bcrypt]>=1.7.4
+pynacl==1.3.0; python_version <= '3.7'
+pynacl==1.4.0; python_version == '3.8'
+pynacl>=1.5.0; python_version >= '3.9'
```

Comment:
Was ensuring minimizing changes per version of Python, will use suggestion

Reply:
Rule of thumb.
If there's a need to upgrade, we should upgrade to what is the minimum at the time of change.
Older python version requirements should follow, unless impossible because it's not supported.